src/main/scala/cromiam/server/config/CromIamServerConfig.scala

```diff
@@ -10,15 +10,17 @@ import scala.collection.JavaConverters._
 import scala.util.{Failure, Success, Try}
 import cromiam.server.config.CromIamServerConfig._
 
-final case class CromIamServerConfig(cromIamConfig: CromIamConfig, cromwellConfig: ServiceConfig, samConfig: ServiceConfig)
+final case class CromIamServerConfig(cromIamConfig: CromIamConfig, cromwellConfig: ServiceConfig,
```

Comment:
IntelliJ disagrees on your definition of "short". 😉 

![screen shot 2017-07-17 at 2 16 34 pm](https://user-images.githubusercontent.com/791985/28283142-cecad5a6-6afa-11e7-8cb4-b2834024ae3c.png)

If someone provides a number, _any_ number, I'll create a new config that uses this new margin besides the default. Still hoping we switch to scalafmt at some point, and if you don't give it a number, it picks "80", instead of IntelliJ's liberal "120".

In the meantime, happy to switch to one per line. That's actually what scalafmt would do for this "long" line.